package.json

```diff
@@ -48,21 +48,21 @@
     "build-storybook": "storybook build"
   },
   "devDependencies": {
-    "@storybook/addon-essentials": "8.0.0-beta.2",
-    "@storybook/addon-interactions": "8.0.0-beta.2",
-    "@storybook/addon-links": "8.0.0-beta.2",
-    "@storybook/blocks": "8.0.0-beta.2",
-    "@storybook/components": "8.0.0-beta.2",
-    "@storybook/core-events": "8.0.0-beta.2",
-    "@storybook/manager": "8.0.0-beta.2",
-    "@storybook/manager-api": "8.0.0-beta.2",
-    "@storybook/preview": "8.0.0-beta.2",
-    "@storybook/preview-api": "8.0.0-beta.2",
-    "@storybook/react": "8.0.0-beta.2",
-    "@storybook/react-vite": "8.0.0-beta.2",
+    "@storybook/addon-essentials": "8.0.0",
```

Comment:
shouldn't the deps be ^8.0.0 instead?

Reply:
This is actually on purpose, originally discussed here https://github.com/storybookjs/addon-kit/pull/59

tl;dr by pinning to 8.0.0 (similar to what we did for 7.0.0) we're encouraging addon authors to support the whole major version range instead of just the latest minor release. If they specifically need an API from a later minor, they can upgrade to that intentionally.

Reply:
Got it! But doesn't that mean they won't notice issues in CI or locally? If 8.1.0 releases something that ends up affecting the addon (which we know can happen), then they won't know. This would make more sense if we set up a nightly CI check that uses npx storybook@latest upgrade and then runs yarn build and optionally yarn test-storybook

Reply:
I see what you mean, it would be ideal to support "both", I don't feel like adding that CI workflow now, since I don't think it's that simple to test an addon, given they are often manager functionality which can't be represented in a story.

I'd rather optimize for addons to support a whole major range, than I'd optimize for addons to catch bugs/regression we introduce in a Storybook minor release.